pkg/system/stat_netbsd.go

```diff
@@ -0,0 +1,13 @@
+package system // import "github.com/docker/docker/pkg/system"
+
+import "syscall"
+
+// fromStatT converts a syscall.Stat_t type to a system.Stat_t type
+func fromStatT(s *syscall.Stat_t) (*StatT, error) {
+	return &StatT{size: s.Size,
+		mode: uint32(s.Mode),
```

Comment:
Looks like this conversion is redundant on netbsd, or did you add on purpose? (if so, perhaps this requires a `//nolint:unconvert`)